site/app/libraries/Core.php

```diff
@@ -730,6 +730,23 @@ public function isWebLoggedIn(): bool {
         return $logged_in;
     }
 
+    /**
+     * Check if we have a saved cookie session with the dropdown states for each of the instructor
+     * polls sections (today, old, future), otherwise sets it to the default dropdown states (true, false, false).
+     *
+     * @return array list of three booleans for dropdown states
+     */
+    public function getPollsDropdownStates(): array {
```

Comment:
Maybe I should put this code directly in PollView.php .. I put it in Core.php because I saw other cookie functions here , but maybe it's unnecessary.

Reply:
Yeah I would add this in the PollController and pass the information onto the view.
You can also have a dictionary which is easier to understand than an array of booleans:
```php
$ret = [
   'today' => true
   'old' => false
   'future' => false
];
```